### ISSUE TITLE
Non-underscore syntax supported for ActionView form helpers?

### ISSUE THREAD

**Issue by bluz71** [loomchild/theo-rails#7]
Hello,

The Theo README lists the following [`<form>`](https://github.com/loomchild/theo-rails?tab=readme-ov-file#forms) helpers syntax:

```erb
<_form-with model%="widget" data-turbo-confirm="Are you sure?">
  <div>
    <_label name="name" />
    <_text-field name="name" />
  </div>

  <div>
    <_label name="size" />
    <_select name="size" options%="['Big', 'Small']" />
  </div>

  <_submit value="Create" />
</_form-with>
```

However, as [I noted in this post](https://github.com/loomchild/theo-rails/issues/5#issuecomment-2458605545) the `<_some-partial>` syntax breaks the standard HTML parser of [Prettier](https://prettier.io).

Question, does Theo support a more HTML-compatible syntax for Rails ActionView helpers?

Basically, I would like to use Theo if I can totally avoid the `<_some-tage-name>` underscore syntax.

Note, I am not saying to get rid of the underscore syntax, just wondering if a Prettier-compatible syntax is available?

Best regards.

**Comment by loomchild** [comment]
Yeah, the JSX-style PascalCase syntax is fully supported and equivalent of _underscore syntax, you can call partials with it. When I'll find the time, I will make it clearer in the README.

Does it answer your question? Or are you asking to have a new way of calling helpers (and not partials)?

**Comment by bluz71** [comment]
So PascalCase is supported for Form Helpers?

For example ‘FormWith‘?
